Places/kumari.md

```diff
@@ -0,0 +1,7 @@
+#### Places to eat
+ - Best bakery
+ - India Cafe
```

Comment:
What is your favorite dish here? Just was in India last month and had tons of great food.